include/bluetooth/bluetooth.h

```diff
@@ -980,6 +980,21 @@ struct bt_le_per_adv_sync_synced_info {
 
 	/** True if receiving periodic advertisements, false otherwise. */
 	bool recv_enabled;
+
+	/**
+	 * @brief Service Data provided by the peer when sync is transferred
```

Comment:
Is it use-case specific? How should the value be interpretet?

Reply:
It is; it's application specific data. The core spec doesn't state any way it should be used; might be used to contain a 16-bit UUID for the application to determine the context of the PAST.